packages/cli/package.json

```diff
@@ -116,7 +116,7 @@
     "zx": "7.2.3"
   },
   "scripts": {
-    "prisma": "ts-node src/bin.ts",
+    "prisma": "tsx src/bin.ts",
```

Comment:
@Jolg42 is this change ok? 

Without this change, one enters ESM rabbit hole.

Reply:
We already use tsx already, we're very happy with it!

Reply:
Anyway, this is only an internal change for a convenience script